src/clarity-angular/datagrid/datagrid.html

```diff
@@ -36,7 +36,7 @@
 
     <!-- Custom placeholder overrides the default empty one -->
     <ng-content select="clr-dg-placeholder"></ng-content>
-    <clr-dg-placeholder *ngIf="!placeholder"></clr-dg-placeholder>
+    <clr-dg-placeholder *ngIf="!placeholder && items.displayed.length == 0"></clr-dg-placeholder>
```

Comment:
This is not the behavior we want. It makes the last page jump when there are fewer items on it, which is the opposite of what we want.